test/Maestro.ScenarioTests/MaestroScenarioTestBase.cs

```diff
@@ -949,4 +950,14 @@ protected async Task<bool> ValidateGithubMaestroCheckRunsSuccessful(string targe
 
         return true;
     }
+
+    protected static string GetTestChannelName([CallerMemberName] string testName = "")
+    {
+        return $"{testName}_{Guid.NewGuid().ToString().Substring(0, 16)}";
+    }
+
+    protected static string GetTestBranchName([CallerMemberName] string testName = "")
+    {
+        return $"{testName}_{Guid.NewGuid().ToString().Substring(0, 16)}";
+    }
```

Comment:
@dkurepa maybe this is better?

```suggestion
    protected static string GetTestChannelName([CallerMemberName] string testName = "")
    {
        return $"c{testName}_{Guid.NewGuid().ToString().Substring(0, 16)}";
    }

    protected static string GetTestBranchName([CallerMemberName] string testName = "")
    {
        return $"b{testName}_{Guid.NewGuid().ToString().Substring(0, 16)}";
    }
```

Reply:
Good idea, will be easier to differentiate them in the logs